### PR TITLE
Add aliases for ns, ms, os, cs shorthand commands supported by many servers to message services easily

### PR DESCRIPTION
**Pull request by FeepingCreature**
Right now, you have to either do `/quote cs ...` or `/msg chanserv ...`, raising potential for leaking information via typos.

Tested on Esper, Freenode and Libera.

### CODE REVIEW

File: src/irc/core/irc-commands.c [hunk 1]
```diff
@@ -1045,6 +1045,11 @@ void irc_commands_init(void)
 	command_bind_irc("knockout", NULL, (SIGNAL_FUNC) cmd_knockout);
 	command_bind_irc("setname", NULL, (SIGNAL_FUNC) command_1self);
 	command_bind_irc("server purge", NULL, (SIGNAL_FUNC) cmd_server_purge);
+	/* common shorthands for MSG NickServ, ChanServ, OperServ, MemoServ :parameter */
+	command_bind_irc("ns", NULL, (SIGNAL_FUNC) command_self);
+	command_bind_irc("cs", NULL, (SIGNAL_FUNC) command_self);
+	command_bind_irc("os", NULL, (SIGNAL_FUNC) command_self);
+	command_bind_irc("ms", NULL, (SIGNAL_FUNC) command_self);
```
Comment (vague666) [inline src/irc/core/irc-commands.c:1052]: You need to unbind the commands too, but I'm wondering if it wouldn't be better to add those as default aliases?

Comment (vague666) [reply src/irc/core/irc-commands.c:1052]: Place to add the aliases: https://github.com/irssi/irssi/blob/master/irssi.conf
But lets let the maindevs have the final say

Comment (FeepingCreature) [reply src/irc/core/irc-commands.c:1052]: Added unbind calls for now, until someone tells me how to add default aliases. :)

**Comment by ailin-nemui** [comment]
Thanks for this pull request, however I believe that the Irssi project prefers to do these commands yourself. Every user can set up their own aliases for example `/alias ns quote ns`
@irssi/developers ?

**Comment by vague666** [comment]
I don't mind adding default aliases, those commands are often referenced in support articles so it would be good if irssi knew what to do with them without the user having to ask for help when stuff fail

**Comment by FeepingCreature** [comment]
Switched to aliases.

**Comment by dequis** [comment]
I'm also ok with these as aliases

**Pull request merged**